tensorflow/core/kernels/mkl_softmax_op.cc

```diff
@@ -62,7 +63,26 @@ class MklSoftmaxOp : public OpKernel {
                               : src_tensor.shape();
       auto src_dims = TFShapeToMklDnnDims(src_tf_shape);
       auto output_dims = src_dims;
-
+      memory::format layout_type;
+      switch (input_dims) {
+        case 1:
+          layout_type = memory::format::x;
```

Comment:
Could you please add comments for the 5 cases here explaining what these comments are or giving their full names.

Thanks!

Reply:
Thank you for fixing the regression test failure! Looks great. Just one small request.

Reply:
I have added some comments for the data format. Please let me know if this is ok.